### PR TITLE
[13.0][FIX+imp] hr_expense_invoice: Set empty tax_ids when create_invoice wizard + Remove expense_create_invoice wizard according to sheet

### PR DESCRIPTION
**Pull request by victoralmau**
[FIX] Set empty `tax_ids` when create_invoice wizard.
[IMP] Remove expense_create_invoice wizard according to sheet

Please @pedrobaeza and @joao-p-marques can you review it?

@Tecnativa TT28869

### CODE REVIEW

**Comment by pedrobaeza** [comment]
Why didn't you cherry-pick my v12 commit?

**Comment by victoralmau** [comment]
It's different about https://github.com/OCA/hr/commit/3bf221e25e735be2c9cb341236a8310cdad754bf (although it's need to apply 12.0 too)

**Comment by pedrobaeza** [comment]
But the other exists in v13?

**Comment by victoralmau** [comment]
Yes, in 13.0 exists in https://github.com/OCA/hr-expense/commit/95f1cff925afc61bc4aea302ba474248660f39c2

**Comment by pedrobaeza** [comment]
OK, are you going to add the constraint?

**Comment by victoralmau** [comment]
> OK, are you going to add the constraint?

IMO is not necessary.

File: hr_expense_invoice/wizard/expense_create_invoice.py [hunk 1]
```diff
@@ -62,5 +62,5 @@ def create_invoice(self):
         invoice = (
             self.env["account.move"].with_context(type="purchase").create(invoice_vals)
         )
-        self.expense_ids.write({"invoice_id": invoice.id})
+        self.expense_ids.write({"invoice_id": invoice.id, "tax_ids": False})
```
Comment (pedrobaeza) [inline hr_expense_invoice/wizard/expense_create_invoice.py]: What about the final amount? Maybe it has been changed.

Comment (victoralmau) [reply hr_expense_invoice/wizard/expense_create_invoice.py]: Is not possible because total of sheet is computed according to lines (expenses).

Comment (pedrobaeza) [reply hr_expense_invoice/wizard/expense_create_invoice.py]: Not total of sheet, but the line.

I see this very weird... Are you saying that the invoice is assigned to all the lines? That's a mistake IMO.

Comment (victoralmau) [reply hr_expense_invoice/wizard/expense_create_invoice.py]: This is done through a wizard when the lines have been selected, so it is possible to assign the same invoice generated to several lines and then, the total cannot be assigned correctly.

Comment (pedrobaeza) [reply hr_expense_invoice/wizard/expense_create_invoice.py]: At least a constraint should be added to raise the error if the total amount doesn't match. If not, there will be problems. @kittiu what is your use case for adding such weird wizard?

Comment (kittiu) [reply hr_expense_invoice/wizard/expense_create_invoice.py]: With my blurry memory. In v8, we are used to a module that create invoice from expense.
But to be frank, nowadays we didn't use it either. Please feel free to remove. It is erroneous to have too many ways to do the same.

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**